pkg/apis/devops/v1alpha2/register.go

```diff
@@ -271,27 +271,27 @@ func addWebService(c *restful.Container) error {
 		Param(webservice.QueryParameter("branch", "branch ").
 			Required(false).
 			DataFormat("branch=%s")).
-		Returns(http.StatusOK, RespOK, []devops.PipelineRun{}).
-		Writes([]devops.PipelineRun{}))
+		Returns(http.StatusOK, RespOK, []devops.PipeBranchRun{}).
+		Writes([]devops.PipeBranchRun{}))
 
 	// match Jenkins api "/blue/rest/organizations/jenkins/pipelines/{projectName}/{pipelineName}/branches/{branchName}/runs/{runId}/"
 	webservice.Route(webservice.GET("/devops/{projectName}/pipelines/{pipelineName}/branches/{branchName}/runs/{runId}").
-		To(devopsapi.GetPipelineRun).
+		To(devopsapi.GetPipeBranchRun).
 		Metadata(restfulspec.KeyOpenAPITags, tags).
-		Doc("Get DevOps Pipelines run.").
+		Doc("Get DevOps Pipelines run in branch.").
 		Param(webservice.PathParameter("pipelineName", "pipeline name")).
 		Param(webservice.PathParameter("projectName", "devops project name")).
 		Param(webservice.PathParameter("branchName", "pipeline branch name")).
 		Param(webservice.PathParameter("runId", "pipeline runs id")).
 		Param(webservice.QueryParameter("start", "start").
 			Required(false).
 			DataFormat("start=%d")).
-		Returns(http.StatusOK, RespOK, devops.PipelineRun{}).
-		Writes(devops.PipelineRun{}))
+		Returns(http.StatusOK, RespOK, devops.PipeBranchRun{}).
+		Writes(devops.PipeBranchRun{}))
 
 	// match Jenkins api "/blue/rest/organizations/jenkins/pipelines/{projectName}/{pipelineName}/branches/{branchName}/runs/{runId}/nodes"
 	webservice.Route(webservice.GET("/devops/{projectName}/pipelines/{pipelineName}/branches/{branchName}/runs/{runId}/nodes").
-		To(devopsapi.GetPipelineRunNodes).
+		To(devopsapi.GetBranchPipeRunNodes).
```

Comment:
what's the difference between `BranchPipe` and `PipeBranch`

Reply:
`BranchPipe` mean all pipeline under the branch, and the `PipeBranch` mean all barnch under the pipeline.

Reply:
ok, so what do you think of `GetPipelinesByBranch` `GetBranchesByPipeline`

Reply:
ok